src/main/java/strategy/impl/OperationStrategyImpl.java

```diff
@@ -0,0 +1,20 @@
+package strategy.impl;
```

Comment:
The package naming convention usually follows the domain name of the company in reverse, followed by the project name and then the module. For example, `com.mycompany.myproject.strategy.impl`. Ensure that the package naming follows your organization's standards.